backend/dal/Configuration/PropertyConfiguration`.cs

```diff
@@ -14,10 +14,10 @@ public abstract class PropertyConfiguration<TBase> : BaseEntityConfiguration<TBa
         public override void Configure(EntityTypeBuilder<TBase> builder)
         {
             builder.Property(m => m.ProjectNumber).HasMaxLength(50);
-            builder.Property(m => m.Name).HasMaxLength(150);
+            builder.Property(m => m.Name).HasMaxLength(250);
             builder.Property(m => m.Description).HasMaxLength(2000);
-            builder.Property(m => m.Latitude).IsRequired();
-            builder.Property(m => m.Longitude).IsRequired();
+            builder.Property(m => m.Location).HasColumnType("GEOGRAPHY").IsRequired();
```

Comment:
Should this be an enum/constant?

Reply:
No it's a string for a specific data type within MS SQL.